app/views/documents/form_partials/_validate_document.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-form-group" id="validate-document">
+  <% if form.object.errors.any? %>
+    <% form.object.errors.each do |error| %>
```

Comment:
Missed this first run-through, but just noticed this prints all errors on any field on the @document object, whereas I think you only want to print the errors related to the validated and invalidated_document_reason fields?

An example of the potential bug here: If there is an error on the `numbers` field, it will create an error message on line 4 of _edit_and_upload.html.erb *and* here.